app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -239,8 +241,25 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
                         .zoom(ZOOM_LEVEL)
                         .build();
                 mapBoxMap.setCameraPosition(cameraPosition);
-            });
 
+                ScaleBarPlugin scaleBarPlugin = new ScaleBarPlugin(mapView, mapBoxMap);
+                int color = isDarkTheme ? R.color.bottom_bar_light : R.color.bottom_bar_dark;
+                //Default text size for scalebar params
+                final float SIZE = new TextView(getContext()).getTextSize() - 1;
```

Comment:
I kind of glanced over this because I thought I already saw review comments but I would caution against this.

Resolve a dimen value that is appropriate here, you can use `context.getResources().getDimensionPixelSize(R.dimen.some_dimen)` and that will give you different sizes depending on the density of the screen. All of these should be like that and the math should be avoided. Constructing a textview to find out the default value of text size is putting the donkey before the cart, a little bit

Reply:
Sure!! I understand, will do that.